### PR TITLE
Use eslint with loopback config

### PR DESCRIPTION
**Pull request by bajtos**
Drop jshint and jscs in favour of eslint. Fix linting issues.

See also https://github.com/strongloop/eslint-config-loopback/pull/1

Connect to strongloop/loopback#1749

@superkhau please review
/cc @strongloop/loopback-dev

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "loopback"
+}
```
Comment (superkhau) [inline .eslintrc]: can this change to .eslintrc.yaml? and i assume with yaml syntax inside?

File: test/executor.test.js [hunk 2]
```diff
@@ -785,11 +786,11 @@ describe('executor', function() {
...
-              phase: 'auth:before'
-            }
-          }
-        ]
-      }
+              phase: 'auth:before',
+            },
+          },
+        ],
+      },
```
Comment (superkhau) [inline test/executor.test.js]: lol all these dangle comma's look painful.

**Comment by superkhau** [comment]
LGTM

**Comment by bajtos** [comment]
@slnode test please

**Pull request merged**